phpBB/phpbb/db/driver/mssql_odbc.php

```diff
@@ -289,6 +289,14 @@ function sql_nextid()
 		return false;
 	}
 
+	/**
+	 * {@inheritdoc}
+	 */
+	public function sql_last_inserted_id()
+	{
+		return $this->sql_nextid();
```

Comment:
I would actually move the logic from `sql_nextid()` to this and call `sql_last_inserted_id()` from `sql_nextid()`. This will make it way easier to later remove `sql_nextid()`.